### PR TITLE
Update release notes for SIG-Cluster-Lifecycle

### PR DESCRIPTION
**Pull request by luxas**
PTAL @jdumars @calebamiles @radhikapc @kubernetes/sig-cluster-lifecycle-pr-reviews @andrewsykim @wlan0

### CODE REVIEW

**Comment by andrewsykim** [comment]
lgtm for out-of-tree cloud provider related changes

File: release-1.8/release_notes_draft.md [hunk 1]
```diff
@@ -211,19 +229,15 @@ kind.
...
+
+* [alpha] The Conformance e2e test suite now passes on the arm, arm64 and ppc64le platforms as well ([#288](https://github.com/kubernetes/features/issues/288), [@luxas](https://github.com/luxas), [@mkumatag](https://github.com/mkumatag), [@ixdy](https://github.com/ixdy))
 
 #### Cloud Providers
-* [beta] Support for out-of-tree and out-of-process cloud providers, a.k.a pluggable providers, has been promoted from alpha to beta. ([#88](https://github.com/kubernetes/features/issues/88), [@wlan0](https://github.com/wlan0))
 
-#### DaemonSet
-* [beta] DaemonSet upgrades can be achieved via a start-then-kill update strategy. ([#373](https://github.com/kubernetes/features/issues/373), [@aaronlevy](https://github.com/aaronlevy), [@diegs](https://github.com/diegs))
+* [alpha] Support for out-of-tree and out-of-process cloud providers, a.k.a pluggable providers, has been significantly improved. Creating an out-of-code cloud provider is now easier and includes more features. ([#88](https://github.com/kubernetes/features/issues/88), [@wlan0](https://github.com/wlan0))
```
Comment (tamalsaha) [inline release-1.8/release_notes_draft.md]: Should this call out to the limitations of current implementations? For example, is https://github.com/kubernetes/kubeadm/issues/408 solved in 1.8 ? 

This description makes it sound like it is ready.

File: release-1.8/release_notes_draft.md [hunk 2]
```diff
@@ -197,12 +208,19 @@ kind.
...
 
 * [beta] A new `upgrade` subcommand allows you to automatically upgrade a self-hosted cluster created with kubeadm. ([#296](https://github.com/kubernetes/features/issues/296), [@luxas](https://github.com/luxas))
 
+* [alpha] It is now possible to easily create a self-hosted cluster with `kubeadm init` by enabling the feature (`--feature-gates=SelfHosting=true`) ([#296](https://github.com/kubernetes/features/issues/296), [@luxas](https://github.com/luxas))
+   * **NOTE:** Self-hosting will be the default way to host the control plane in v1.9
+
+* [alpha] A new `phase` subcommand supports performing only subtasks of the full `kubeadm init` flow. Combined with fine-grained configuration, kubeadm is now more easily consumable by higher-level provisioning tools like kops or GKE. ([#356](https://github.com/kubernetes/features/issues/356), [@luxas](https://github.com/luxas))
+   * **NOTE:** This command is currently staged under `kubeadm alpha phase`, will graduate to top-level in a future release
+
```
Comment (kad) [inline release-1.8/release_notes_draft.md]: Maybe let's add line also about ability to install easily builds from master branch by specifying "ci/latest" ?

Comment (luxas) [reply release-1.8/release_notes_draft.md]: That's too detailed I'm afraid, we don't do that in other places as well.
Depending on what we decide, we might have a section below for smaller changes.

This section is only for issues tracked in kubernetes/features

File: release-1.8/release_notes_draft.md (hunk 1 above)
Comment (luxas) [reply release-1.8/release_notes_draft.md]: That issue will actually make it in v1.8 I think :smile:
Any suggestion how to reword?

Comment (tamalsaha) [reply release-1.8/release_notes_draft.md]: > That issue will actually make it in v1.8 I think . Great!

> Any suggestion how to reword?
Something like "For future todos, check here (link to github issue). Essentially, some way to get a sense what is yet not possible and where are those worked on. For example, I read that Volume support is not possible by external CCM in one of the threads about DO CCM.

**Comment by roberthbailey** [comment]
lgtm

thanks for fixing these up.

File: release-1.8/release_notes_draft.md [hunk 3]
```diff
@@ -211,19 +229,15 @@ kind.
...
+* [alpha] The Conformance e2e test suite now passes on the arm, arm64 and ppc64le platforms as well ([#288](https://github.com/kubernetes/features/issues/288), [@luxas](https://github.com/luxas), [@mkumatag](https://github.com/mkumatag), [@ixdy](https://github.com/ixdy))
 
 #### Cloud Providers
-* [beta] Support for out-of-tree and out-of-process cloud providers, a.k.a pluggable providers, has been promoted from alpha to beta. ([#88](https://github.com/kubernetes/features/issues/88), [@wlan0](https://github.com/wlan0))
 
-#### DaemonSet
-* [beta] DaemonSet upgrades can be achieved via a start-then-kill update strategy. ([#373](https://github.com/kubernetes/features/issues/373), [@aaronlevy](https://github.com/aaronlevy), [@diegs](https://github.com/diegs))
```
Comment (diegs) [inline release-1.8/release_notes_draft.md]: Thanks for removing this, we will try to push it in as early in the 1.8 cycle as possible.

**Comment by luxas** [comment]
@roberthbailey any blocker for merging this or can I click the button :)?

File: release-1.8/release_notes_draft.md [hunk 4]
```diff
@@ -90,6 +90,10 @@ fundamental aspect of a secure cluster.
 ### Auth
 - With the introduction of RBAC v1, the RBAC v1alpha1 API group has been deprecated.
 
+### Cluster Lifecycle
+
+- The auto-detect behavior of the kubelet's `--cloud-provider` flag is deprecated. In v1.9, the default will be `""`, which means no built-in cloud provider extension will be enabled by default. [PR #51312](https://github.com/kubernetes/kubernetes/pull/51312) and [announcement](https://groups.google.com/forum/#!topic/kubernetes-dev/UAxwa2inbTA)
```
Comment (radhikapc) [inline release-1.8/release_notes_draft.md]: v1.9 or v1.8 ?

File: release-1.8/release_notes_draft.md [hunk 5]
```diff
@@ -128,6 +132,15 @@ kind.
  in place of [Opaque Integer Resources](https://v1-6.docs.kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#opaque-integer-resources-alpha-feature).
  Extended resources cannot be overcommitted, so request and limit must be equal
  if both are present in a container spec.
+ - The default Bootstrap Token created with `kubeadm init` v1.8 expires
+ and is deleted after 24 hours by default to limit the exposure of the
+ valuable credential. It is straightforward to create a new Bootstrap
+ Token with `kubeadm token create` afterwards, or alternatively, you can
```
Comment (radhikapc) [inline release-1.8/release_notes_draft.md]: You can create a new Bootstrap Token with `kubeadm token create` or  make the default token permanently valid by specifying `--token-ttl 0` to `kubeadm init`. The default token can later be deleted with `kubeadm token delete`.

File: release-1.8/release_notes_draft.md (hunk 4 above)
Comment (radhikapc) [inline release-1.8/release_notes_draft.md]: v1.8 or v1.9 ?  If v1.9, what's the default behavior in 1.8

File: release-1.8/release_notes_draft.md [hunk 6]
```diff
@@ -128,6 +132,15 @@ kind.
...
+ specify `--token-ttl 0` to `kubeadm init` to make the default token valid
+ forever (it can be deleted afterwards with `kubeadm token delete`, though)
```
Comment (radhikapc) [inline release-1.8/release_notes_draft.md]: You can create a new Bootstrap Token with `kubeadm token create` or 
 make the default token permanently valid by setting `--token-ttl 0` to `kubeadm init`. The default token can later be deleted with `kubeadm token delete`.

File: release-1.8/release_notes_draft.md [hunk 7]
```diff
@@ -197,12 +208,19 @@ kind.
...
 
 ### **Cluster Lifecycle**
+
 #### kubeadm
-* [beta] A new `phase` subcommand supports performing only subtasks of the full `kubeadm init` flow. Combined with fine-grained configuration, kubeadm is now more easily consumable by higher-level provisioning tools like kops or GKE. ([#356](https://github.com/kubernetes/features/issues/356), [@luxas](https://github.com/luxas), [@justinsb](https://github.com/justinsb))
 
 * [beta] A new `upgrade` subcommand allows you to automatically upgrade a self-hosted cluster created with kubeadm. ([#296](https://github.com/kubernetes/features/issues/296), [@luxas](https://github.com/luxas))
 
+* [alpha] It is now possible to easily create a self-hosted cluster with `kubeadm init` by enabling the feature (`--feature-gates=SelfHosting=true`) ([#296](https://github.com/kubernetes/features/issues/296), [@luxas](https://github.com/luxas))
```
Comment (radhikapc) [inline release-1.8/release_notes_draft.md]: A self-hosted cluster can be easily created with `kubeadm init`. Enable the feature by setting the `SelfHosting` flag to true: `--feature-gates=SelfHosting=true`

File: release-1.8/release_notes_draft.md [hunk 8]
```diff
@@ -197,12 +208,19 @@ kind.
...
+   * **NOTE:** Self-hosting will be the default way to host the control plane in v1.9
```
Comment (radhikapc) [inline release-1.8/release_notes_draft.md]: v1.9 or v1.8 ?

File: release-1.8/release_notes_draft.md [hunk 9]
```diff
@@ -197,12 +208,19 @@ kind.
...
+
+* [alpha] A new `phase` subcommand supports performing only subtasks of the full `kubeadm init` flow. Combined with fine-grained configuration, kubeadm is now more easily consumable by higher-level provisioning tools like kops or GKE. ([#356](https://github.com/kubernetes/features/issues/356), [@luxas](https://github.com/luxas))
+   * **NOTE:** This command is currently staged under `kubeadm alpha phase`, will graduate to top-level in a future release
```
Comment (radhikapc) [inline release-1.8/release_notes_draft.md]: This command is currently staged under `kubeadm alpha phase` and will be graduated to top-level in a future release.

File: release-1.8/release_notes_draft.md [hunk 10]
```diff
@@ -211,19 +229,15 @@ kind.
 
 #### Cluster Discovery/Bootstrap
 
-* [beta] The authentication and verification mechanism called Bootstrap Tokens has been added to the core API, which can be used to easily add new members to a cluster. ([#130](https://github.com/kubernetes/features/issues/130), [@luxas](https://github.com/luxas), [@jbeda](https://github.com/jbeda)).
-
-#### Addons
-* [alpha] A new system addon manager is available that is aiming to improve the downsides of the existing `kube-addons.sh` manager. ([#18](https://github.com/kubernetes/features/issues/18), [@justinsb](https://github.com/justinsb))
+* [beta] The authentication and verification mechanism called Bootstrap Tokens has been added to the core API, which can be used to easily add new node identities to a cluster. ([#130](https://github.com/kubernetes/features/issues/130), [@luxas](https://github.com/luxas), [@jbeda](https://github.com/jbeda)).
```
Comment (radhikapc) [inline release-1.8/release_notes_draft.md]: The authentication and verification mechanism called Bootstrap Tokens has been added to the core API. Use Bootstrap Tokens to add new node identities to a cluster easily.

File: release-1.8/release_notes_draft.md [hunk 11]
```diff
@@ -211,19 +229,15 @@ kind.
...
 
 #### Multi-platform
-* [beta] Kubernetes now has automated continuous-integration tests against all of our supported platforms (amd64, armhfp, aarch64, ppc64le), to ensure that it continues to work on these platforms. It's also possible to run clusters with nodes of mixed architectures. ([#288](https://github.com/kubernetes/features/issues/288), [@luxas](https://github.com/luxas), [@mkumatag](https://github.com/mkumatag), [@ixdy](https://github.com/ixdy))
+
+* [alpha] The Conformance e2e test suite now passes on the arm, arm64 and ppc64le platforms as well ([#288](https://github.com/kubernetes/features/issues/288), [@luxas](https://github.com/luxas), [@mkumatag](https://github.com/mkumatag), [@ixdy](https://github.com/ixdy))
```
Comment (radhikapc) [inline release-1.8/release_notes_draft.md]: The Conformance e2e test suite now passes on the arm, arm64, and ppc64le platforms.

File: release-1.8/release_notes_draft.md (hunk 1 above)
Comment (radhikapc) [inline release-1.8/release_notes_draft.md]: Support for the pluggable providers, out-of-tree and out-of-process cloud providers, has been significantly improved.

File: release-1.8/release_notes_draft.md (hunk 4 above)
Comment (luxas) [reply release-1.8/release_notes_draft.md]: auto-detect is still the default in v1.8; that behavior is deprecated and will change in v1.9

File: release-1.8/release_notes_draft.md (hunk 8 above)
Comment (luxas) [reply release-1.8/release_notes_draft.md]: v1.9; we want to do a heads up already

File: release-1.8/release_notes_draft.md (hunk 5 above)
Comment (luxas) [reply release-1.8/release_notes_draft.md]: fixed

File: release-1.8/release_notes_draft.md (hunk 7 above)
Comment (luxas) [reply release-1.8/release_notes_draft.md]: fixed.

File: release-1.8/release_notes_draft.md (hunk 9 above)
Comment (luxas) [reply release-1.8/release_notes_draft.md]: fixed

File: release-1.8/release_notes_draft.md (hunk 10 above)
Comment (luxas) [reply release-1.8/release_notes_draft.md]: fixed

File: release-1.8/release_notes_draft.md (hunk 11 above)
Comment (luxas) [reply release-1.8/release_notes_draft.md]: fixed

File: release-1.8/release_notes_draft.md (hunk 1 above)
Comment (luxas) [reply release-1.8/release_notes_draft.md]: fixed

**Comment by luxas** [comment]
@radhikapc @idvoretskyi @roberthbailey WDYT?

**Comment by radhikapc** [comment]
lgtm

**Pull request merged**